Rules/CommonScripts/ChatCommands/PlayerCommands.as

```diff
@@ -95,6 +95,32 @@ class TeamCommand : ChatCommand
 		}
 
 		blob.server_setTeamNum(team);
+		
+		// convert items in inventory
+		CInventory@ inv = blob.getInventory();
+		if (inv !is null)
+		{
+			for (int i = 0; i < inv.getItemsCount(); i++)
+			{
+				CBlob@ item = inv.getItem(i);
+				
+				if (item !is null)
```

Comment:
Why no such `SetTeamToCarrier.as` check here?

Reply:
I can't think of a reason why not, so I included it.